### ISSUE TITLE
i opened the program and i can't click add download. it worked yesterday

### ISSUE THREAD

**Issue by Latackty** [NickvisionApps/Parabolic#851]
![Screenshot from 2024-10-02 22-32-34](https://github.com/user-attachments/assets/07b41618-9c02-4eae-98eb-6bdfe0cf2d65)

**Comment by nlogozzo** [comment]
Are you using snap or flatpak?

Are you using Wireguard or any other VPN service?

**Comment by Francehelder** [comment]
Same issue here!

> Could you run the following command and provide the output for me:
> 
> ```
> dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager --print-reply /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Get string:'org.freedesktop.NetworkManager' string:'State'
> ```

![image](https://github.com/user-attachments/assets/59374260-d2a6-4250-8aa1-72e210b5ab3b)

**Comment by Latackty** [comment]
> Are you using snap or flatpak?
> 
> Are you using Wireguard or any other VPN service?
> 
> Can you run the app in the terminal and provide me with the log output?
> 
> Could you run the following command and provide the output for me:
> 
> ```
> dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager --print-reply /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Get string:'org.freedesktop.NetworkManager' string:'State'
> ```

here is what i got from the command:   method return time=1727992889.426631 sender=:1.14 -> destination=:1.311 serial=44747 reply_serial=2
   variant       uint32 70
   and im using the snap version on ubuntu

**Comment by Latackty** [comment]
> dbus-send --system --type=method_call --dest=org.freedesktop.NetworkManager --print-reply /org/freedesktop/NetworkManager org.freedesktop.DBus.Properties.Get string:'org.freedesktop.NetworkManager' string:'State'

method return time=1727993057.713489 sender=:1.14 -> destination=:1.316 serial=45043 reply_serial=2
   variant       uint32 70

**Comment by tlpwka** [comment]
Same thing.
Debian 12. Flatpak version, freshly installed.
No VPN or Wireguard.
`method return time=1727997298.666681 sender=:1.9 -> destination=:1.248 serial=1633 reply_serial=2
   variant       uint32 70`

**Comment by nlogozzo** [comment]
@soumyaDghosh This should be fixed too now, correct?

**Comment by soumyaDghosh** [comment]
Yes, but it needs to be either reinstalled once, or needs to be refreshed once to sync with the store.

**Comment by nlogozzo** [comment]
[V2024.10.1](https://github.com/NickvisionApps/Parabolic/releases/tag/2024.10.1) is here with a fix for this issue :)

*Allow it some time to be available via Snap*
